### ISSUE TITLE
[no-duplicate-attrs] Attribute keys should be checked case-insensitively.

### ISSUE THREAD

**Issue by yeonjuan** [yeonjuan/html-eslint#268]
Because html attribute names are case-insensitive, [no-duplicate-attrs](https://html-eslint.org/docs/rules/no-duplicate-attrs) should also raise a lint error for attribute keys with different cases. ([Case insensitivity in tag names and attribute names](https://w3c.github.io/html-reference/documents.html#case-insensitivity))

[playground](https://html-eslint.org/playground#eyJjb2RlIjoiXG48ZGl2IGlkPVwiMVwiXG4gICAgIElEPVwiMVwiXG4gICAgIElkPVwiMVwiXG4gICAgIGlkPVwiXCJcbiA+PC9kaXY+XG4gICAgXG4iLCJjb25maWciOnsicnVsZXMiOnsiQGh0bWwtZXNsaW50L25vLWR1cGxpY2F0ZS1hdHRycyI6ImVycm9yIn19LCJsYW5ndWFnZSI6Imh0bWwifQ==)
```html
<div id="1"
     ID="1" < -- no error (bug)
     Id="1" < -- no error (bug)
     id=""  < --- error (ok)
 ></div>
```

-
